services/gateway/tests/constants/generateDocs.js

```diff
@@ -337,6 +337,58 @@ const createApiDocsExpectedResponse = {
 			},
 		},
 	},
+	'/blockchain/apps/meta/tokens/supported': {
+		get: {
+			tags: [
+				'Interoperability',
+			],
+			summary: 'Requests blockchain applications off-chain metadata for tokens supported on the specified chainID.',
+			description: 'Returns blockchain applications off-chain metadata for tokens supported on the specified chainID.\n RPC => get.blockchain.apps.meta.tokens.supported',
+			parameters: [
+				{
+					$ref: '#/parameters/chainID',
+				},
+				{
+					$ref: '#/parameters/limit',
+				},
+				{
+					$ref: '#/parameters/offset',
+				},
+				{
+					name: 'sort',
+					in: 'query',
+					description: 'Fields to sort results by.',
+					required: false,
+					type: 'string',
+					enum: [
+						'tokenID:asc',
+						'tokenID:desc',
+					],
+					default: 'tokenID:asc',
+				},
+			],
+			responses: {
+				200: {
+					description: 'Returns a list of blockchain applications off-chain metadata for tokens supported on the specified chainID.',
+					schema: {
+						$ref: '#/definitions/BlockchainAppsTokenMetadataWithEnvelope',
+					},
+				},
+				400: {
+					description: 'Bad request',
+					schema: {
+						$ref: '#/definitions/badRequest',
+					},
+				},
+				404: {
```

Comment:
Do we support 404 for this request?
As per issue #1544 we don't support 404 for any endpoint.

Reply:
Yes, we don't support 404 and the issue you mention will take care of removing 404 from the swagger response.